contribute.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Mozilla IRC iplimit",
+  "description": "Self-service tool for adding IRC exceptions for one's IP address",
+  "repository": {
+    "url": "https://github.com/mozilla/irc-iplimit",
+    "license": "MPL2"
+  },
+  "keywords": [
+    "Python",
+    "Flask",
+    "JSON"
+  ],
+	"participate": {
```

Comment:
This whitespace indent pattern doesn't match the rest of the JSON. It won't cause parsing errors, but it does make it harder to read as a human being.